convert-pth-to-ggml.py

```diff
@@ -126,6 +126,7 @@ def get_n_parts(dim):
             text = tokenizer.id_to_piece(i).replace("\u2581", " ").encode("utf-8")
             fout.write(struct.pack("i", len(text)))
             fout.write(text)
+            fout.write(struct.pack("f", tokenizer.get_score(i)))
```

Comment:
The score is only written for this type of token, but should be written for all.
```suggestion
        
        fout.write(struct.pack("f", tokenizer.get_score(i)))
```

Reply:
omg I broke this while fixing whitespace formatting up 😓